### ISSUE TITLE
Does Grbl-Advanced use the grbl stepper "on" time setting?

### ISSUE THREAD

**Issue by jdeline** [Schildkroet/GRBL-Advanced#81]
This is less of an "issue" and more of a question.  In line number 118 of "stepper.c", the following variable is declared:

uint8_t step_pulse_time;  // Step pulse reset time after step rise

I cannot find where the variable gets assigned, nor where it gets used in the program.  How does GA set the stepper "on" time?

Do you want to create a "Discussions" section in this repository for questions like this?
Thanks!

**Comment by Schildkroet** [comment]
No, this setting isn't used anymore. Step pulse time is a fixed value
https://github.com/Schildkroet/GRBL-Advanced/blob/software/grbl/Stepper.c#L459-L464
